doc/minutes/2022-12-02.md

```diff
@@ -49,15 +54,14 @@ For example, in the case of Cris it would be `template-fillin/reusability`
     -   Dani, Lea, Jesús and Cris will review will review a part of the skeleton
         and elaborate it in maximum 400 characters.
     -   Rober will be the reviewer of the skeleton as a whole.
-    -   It would be necessary to create a branch that comes out of the branch
-        `template-fillin` and that corresponds to our section. For example, in
-        the case of Cris it would be `template-fillin/reusability`
+    -   bifurcaste branch `drafting/template-fillin` to fill in the corresponding 
+        section; e.g., `drafting/reusability` to fill in section "Reusability"
 
 -   **Postponed**: *Approve of the previous minutes*. We left pending for the
```

Comment:
Only "Approve of the previous minutes" is necessary, the rest is "literature" ;)

Reply:
Mind also that this should have its own action point (it's not related to "Skeleton of the proposal")